.github/actions/paths-filter/filters.yml

```diff
@@ -1,18 +1,41 @@
-rust:
+shared: &shared
   - .github/**
+
+rust:
+  - *shared
   - oxidation/**/*.rs
   - oxidation/**/*.toml
   - Cargo.toml
   - Cargo.lock
+
 python:
-  - .github/**
+  - *shared
   - parsec/**
   - tests/**
   - pyproject.toml
   - poetry.lock
   - build.py
+
 rust-ext:
+  - *shared
   - .github/**
   - src/**/*.rs
   - Cargo.toml
   - Cargo.lock
+
+ionic-client: &ionic-client
+  - oxidation/client/src/**
```

Comment:
And what about  `oxidation/bindings/neon/**` & `oxidation/bindings/common` ?

Reply:
`neon` is for electon, so it's not used for the web client, but yes I will add `bindings/common`

Reply:
So `ionic-client` is the common part and `web` the platform specific one.

Why not call it `client-common`, `client-web`, `client-desktop`, `client-android` then ?

Reply:
yes `ionic-client` is the common part

I will rename them then

Reply:
note that the other filter for `electron` & `android` will come later

Reply:
done